README.md

```diff
@@ -76,7 +76,8 @@ The following games are known to integrate GameMode support (meaning they don't
 Other apps which can integrate with GameMode include:
 * [ATLauncher](https://atlauncher.com/downloads) Minecraft launcher
 * [Cemu](https://cemu.info/) Wii U emulator.
-* GNOME Shell ([via extension](https://github.com/gicmo/gamemode-extension)) - indicates when GameMode is active in the top panel.
+* GNOME Shell ([via extension](https://github.com/gicmo/gamemode-extension)) - indicates when GameMode is active in the top panel. (not maintained)
+* [Gnome Shell Extension](https://github.com/trsnaqe/gamemode-shell-extension) - Rewritten and up-to-date GameMode Status Indicator for latest GNOME versions.
```

Comment:
Note that GNOME should be in all caps. See this link: https://foundation.gnome.org/logo-and-trademarks/

Could you simplify the description? It makes sense if we were to keep the previous line, but its too verbose without that context.

Something like "GameMode status indicator extension for GNOME Shell" would be best.

Reply:
Done.